### DISCUSSION TITLE
Selenide opens chromedrivers in the background

### DISCUSSION THREAD

**Discussion by PerfectVertigo** [selenide/selenide#2742] [Q&A]
So I know that selenide handles closing webdrivers on its own, but I still have added Selenide.closeWebDriver() in Aftersuite. But despite that there are several chrome.exe and chromedriver.exe instances created in task manager background process even from one simple test. 
I even tried running cmd command to kill all chrome instances after the test run, it kills drivers opened prior the test run, but new ones are created after the execution is finished, so can't be caught by teardown methods.
My colleagues have the same problem. Without using cmd taskkill in teardown, after the end of the work day, there are hundred of chromedrivers opened in background. Can't seem to find the cause.

I'm using latest version of Selenide

**Comment by PerfectVertigo** [comment]
I've seen --no-sandbox solution, but I have not set any chrome option.

**Comment by asolntsev** [comment]
Seems to be a duplicate of https://github.com/selenide/selenide/issues/2731

It's a [bug in Chrome](https://github.com/selenide/selenide/issues/2731#issuecomment-2084990276), we cannot fix it in Selenide.

**Reply by PerfectVertigo** [reply]
I see. I've checked that, but I don't have --no-sendbox option set. Yet, still have the issue. Weird.
